pkgs/development/tools/workbench/do-not-copy-source-perms.patch

```diff
@@ -0,0 +1,24 @@
+From b5862ec862a1e9a4b01b09f3ca341e57999353c3 Mon Sep 17 00:00:00 2001
+From: hqurve <hqurve@outlook.com>
+Date: Wed, 8 Feb 2023 07:31:04 -0400
+Subject: [PATCH] do not copy source perms
+
+---
+ src/langs/vala/vala.js | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/src/langs/vala/vala.js b/src/langs/vala/vala.js
+index e5ccfcc..dc0b25b 100644
+--- a/src/langs/vala/vala.js
++++ b/src/langs/vala/vala.js
+@@ -12,7 +12,7 @@ export function setup({ data_dir, document }) {
+ 
+   api_file.copy(
+     Gio.File.new_for_path(data_dir).get_child("workbench.vala"),
+-    Gio.FileCopyFlags.OVERWRITE,
++    Gio.FileCopyFlags.OVERWRITE | Gio.FileCopyFlags.TARGET_DEFAULT_PERMS,
```

Comment:
Would be nice to upstream this.

Reply:
Created an upstream PR https://github.com/sonnyp/Workbench/pull/188

Going to use this as a patch for now